google/cloud/spanner/admin/integration_tests/backup_integration_test.cc

```diff
@@ -151,16 +161,19 @@ TEST_F(BackupIntegrationTest, BackupRestore) {
     operation->metadata().UnpackTo(&metadata);
     db_names.push_back(metadata.database());
   }
-  EXPECT_LE(1, std::count(db_names.begin(), db_names.end(), database->name()))
-      << "Database " << database->name()
-      << " not found in the backup operation list.";
+  if (Emulator()) {
+    EXPECT_THAT(db_names, Not(Contains(database->name())));
+  } else {
+    EXPECT_LE(1, std::count(db_names.begin(), db_names.end(), database->name()))
+        << "Database " << database->name()
+        << " not found in the backup operation list.";
+  }
```

Comment:
Seems like the existing test was less of a change detector test?

Reply:
I'm not sure what you mean.  The `<= 1` is because there is a race with when the operation we just started shows up in the list.